pkgs/top-level/aliases.nix

```
@@ -562,6 +562,8 @@ mapAliases ({
   graalvm11-ce = throw "graalvm11-ce has been removed because upstream dropped support to different JDK versions for each GraalVM release. Please use graalvm-ce instead"; # Added 2023-09-26
   graalvm17-ce = throw "graalvm17-ce has been removed because upstream dropped support to different JDK versions for each GraalVM release. Please use graalvm-ce instead"; # Added 2023-09-26
   graalvm19-ce = throw "graalvm19-ce has been removed because upstream dropped support to different JDK versions for each GraalVM release. Please use graalvm-ce instead"; # Added 2023-09-26
+  graalvmCEPackages = graalvmPackages;
+  graalvm-ce = graalvmPackages.graalvm-ce;
```

```suggestion
  graalvmCEPackages = graalvmPackages; # 2024-08-10
```

aliases.nix is for deprecated aliases, so you have to put the date in there as a rough estimate of whether it's okay to replace the alias with a `throw` (see ./maintainers/scripts/remove-old-aliases.py)

graalvm-ce is probably fine to not deprecate? in that case, just put it back to all-packages.nix

---

`graalvm-ce` now lives in `graalvmPackages.graalvm-ce` so  I'm deprecating this old name https://github.com/NixOS/nixpkgs/pull/321026#discussion_r1692035568